.github/CODEOWNERS

```diff
@@ -5,10 +5,12 @@
 **/testing_utils/ @snowflakedb/snowcli @snowflakedb/nade
 src/snowflake/cli/api/utils/ @snowflakedb/snowcli @snowflakedb/nade
 **/test_data @snowflakedb/snowcli @snowflakedb/nade
```

Comment:
line 4 and line 7 should be honored when we introduce our own tests' snapshots in similarly named subdirectories.